engine/src/types.rs

```diff
@@ -236,10 +236,8 @@ impl std::fmt::Debug for QualifiedName {
 /// cxx.
 #[derive(Error, Clone, Debug)]
 pub enum InvalidIdentError {
-    #[error("Union are not supported by autocxx (and their bindgen names have __ so are not acceptable to cxx)")]
+    #[error("Union are not supported by autocxx")]
     Union,
-    #[error("Bitfields are not supported by autocxx (and their bindgen names have __ so are not acceptable to cxx)")]
-    Bitfield,
```

Comment:
I can leave this variant in (and just never construct it) if you don't want a semver break. Let me know.